### DISCUSSION TITLE
Setting id attribute?

### DISCUSSION THREAD

**Discussion by rhavin** [spakin/SimpInkScr#98] [Q&A] [answered]
How can I set the id attribute on a created form?
for example rect((10,10),(20,20),id="foo") ends up with the id=foo in the style section.

**Comment by rhavin** [comment, marked as answer]
OK, never mind, I found it out in the code:


r = rect((x1, y1), (x2, y2))
r._inkscape_obj.set_id("your_id_here")


is there a better way?

**Reply by spakin** [reply]
> is there a better way?

No, I believe what you came up with is the best way.  Thanks for reporting your solution.
